components/src/core/components/Input/switch-input.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../styles';
+@import 'variables';
+
+.oxd-switch-wrapper {
+  & label {
+    cursor: pointer;
+    display: flex;
+    align-content: center;
+    font-family: $oxd-input-control-font-family;
+    color: $oxd-input-control-font-color;
+    font-size: $oxd-input-control-font-size;
+    font-weight: $oxd-input-control-font-weight;
+    width: fit-content;
+    & input[type='checkbox'] {
```

Comment:
hope this selector won't get an issue since we scoped.

Reply:
no issue